platform/viewer/src/routes/Mode/Mode.tsx

```diff
@@ -250,8 +259,9 @@ export default function ModeRoute({
     hangingProtocolService.setActiveProtocols(hangingProtocol);
     mode?.onModeEnter({ servicesManager, extensionManager, commandsManager });
 
-
     const setupRouteInit = async () => {
+      const seriesInstanceUID = query.get('SeriesInstaceUID');
```

Comment:
Grab from the query the actual full list of parameters as the filters, and then exclude the studyInstanceUIDs parameter from it, as that one shouldn't be present.  Something like:
const filters = Object.fromEntries(query.values.filter(([key]) => key!='StudyInstanceUIDs'))
You should play with that in a nodejs command line tool - I may well not have it quite right there.